package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "data-source-helper",
-  "version": "0.1.1",
+  "name": "mendix-data-source-helper",
+  "version": "0.1.2-rc1",
   "description": "Mendix widget data source helper",
```

Comment:
Description sucks...

Reply:
`This module assists with applying multiple restrictions (coming from multiple ) onto the mendix data containers (eg: list-view)`